crawl-ref/source/god-abil.cc

```diff
@@ -5097,6 +5101,7 @@ int pakellas_surge_devices()
     if (!you_worship(GOD_PAKELLAS) || !you.duration[DUR_DEVICE_SURGE])
         return 0;
 
+    // XXX TODO: support djinn here (HP_CASTING, pay_mp, etc)
```

Comment:
I admire the dedication of this TODO but this would require an original dj, upgraded to a time when pakellas existed, then upgraded to current trunk.

(Also, I think this would be solved by my general comment about trying to roll everything into `dec_mp`.)

Reply:
Actually, @ebering keeps threatening to bring Pak back, so this was a warning for him when and if he opts to do so :)